projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/add-to-cart-dialog.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as siteContextSelector from '../../helpers/site-context-selector';
+
+/**
+ * This test checks accessibility concerns on the add to cart modal (PLP Page) using Access Continuum
+ */
+describe('Add to Cart Modal Accessibility', () => {
+  beforeEach(() => {
+    cy.a11yContinuumSetup();
+  });
+
+  context('Add to Cart (PDP)', () => {
+    it('Item Added to Cart Dialog', () => {
+      cy.visit(siteContextSelector.PRODUCT_PATH_2);
+      cy.get('cx-add-to-cart button[type=submit]').click();
+      cy.get('cx-added-to-cart-dialog .cx-modal-content cx-cart-item')
```

Comment:
`cy.get('cx-added-to-cart-dialog .cx-modal-content cx-cart-item')` is fine for making sure that the dialog has loaded first. But then we should run the scope of this test on the entire dialog DOM.
`cy.get('cx-added-to-cart-dialog').a11yRunContinuumTest();`